microsoft/PowerToys
# [PTRun]Remove the timezone plugin

## Summary of the Pull Request

We've been asked to take out the TimeZone plugin for this Release. We're still getting details on what's happening, but it needs to be removed for the time being. Sorry about this! 🤷 


## Validation Steps Performed
Built PowerToys from a clean state after having used the plugin and changed its settings.
Ran the new build to verify if the removal of a plugin caused any crashes if its config data was already on the machine.
